src/Views/login.vue

```diff
@@ -17,18 +15,37 @@ import axios from "axios";
 
   export default {
     name: 'LoginPage',
+    props: ['error', 'bounceto'],
     data () {
       return {
-        firstname: "",
-        lastname: "",
+        username: undefined,
+        password: undefined,
+        errorMessage: this.error,
       }
     },
   methods: {
-    login(username, password) {
+    login() {
       axios.post(process.env.VUE_APP_API_ORIGIN+'/authenticate', {
-        "username": `${username}`,
-        "password": `${password}`
-      }).then(response => console.log(response))
+        "username": this.username,
+        "password": this.password
+      })
+      .then(response => {
+              if (!response || !response.data || !response.data.bearer) {
+                this.errorMessage = 'Bad response from the server'; // this sucks
```

Comment:
Definitely should look into i18n sooner rather than later

Reply:
Good point. Created #42.